src/contrib/tf_op/tvm_dso_ops.cc

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include "tensorflow/core/framework/op.h"
+
+#define REGISTER_TFTVM_OP(n) REGISTER_OP("TvmDsoOp" #n) \
```

Comment:
Could this be improved to use variable number of inputs/outputs rather than hard coded ones?

This is same as: https://github.com/tobegit3hub/tftvm/issues/9

Reply:
> Could this be improved to use variable number of inputs/outputs rather than hard coded ones?
> 
> This is same as: [tobegit3hub/tftvm#9](https://github.com/tobegit3hub/tftvm/issues/9)

This is by limitation of TensorFlow OpKernel class, which only support fixed input number for each op kernel implementation.

Reply:
Yeah, but the input can be a list, which is pretty much variadic:

See this op:
https://github.com/tensorflow/tensorflow/blob/240497c2b2058ed25b09308dd906441882ca7611/tensorflow/compiler/jit/ops/xla_ops.cc#L27

Reply:
That sounds great!

Reply:
Is this fixed?